website/docs/cheatsheets/python/v1/distributed-training.md

```diff
@@ -66,7 +66,7 @@ Simply ensure that you have taken care of the following:
 * Create an `MpiConfiguration` with your desired distribution.
 
 #### Example
-* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/python-sdk/workflows/train/tensorflow/mnist-distributed-horovod)
+* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/workflows/train/tensorflow/mnist-distributed-horovod)
```

Comment:
Hi @konabuta, could you change the links on this page too? They should be already updated in #79.

Reply:
Sorry I missed this file. I modified that points.